cmd/push.go

```diff
@@ -61,7 +61,7 @@ func Push(ctx context.Context) *cobra.Command {
 		Hidden: true,
 		Use:    "push [service]",
 		Short:  "Build, push and redeploy source code to the target app",
-		Args:   utils.MaximumNArgsAccepted(1, "https://www.okteto.com/docs/0.10/reference/cli/#push"),
+		Args:   utils.MaximumNArgsAccepted(1, "https://www.okteto.com/docs/reference/okteto-cli/"),
```

Comment:
There is not docs page for the push command, so we send them to the okteto cli page